optimisation/model.py

```diff
@@ -0,0 +1,51 @@
+import pandas as pd
+import numpy as np
+from sklearn.model_selection import train_test_split
+from sklearn.datasets import make_regression
+
+import numpy 
+
+import tensorflow as tf
+from tensorflow.keras.models import Sequential
+from tensorflow.keras.layers import Dense, ReLU
+
+# Parameters of the data-set
+n_samples = 10000
```

Comment:
@truskovskiyk For model optimization I used another custom model to optimize via pruning due to a lack of documentation about autokeras model optimizations.